packages/pn-personafisica-webapp/sitemap.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+
+    <url>
+        <loc>https://cittadini.notifichedigitali.it/</loc>
+        <lastmod>2024-06-27T15:41:57+00:00</lastmod>
+        <priority>1.00</priority>
+    </url>
+    <url>
+        <loc>https://cittadini.notifichedigitali.it/notifiche</loc>
+        <lastmod>2024-06-27T15:41:57+00:00</lastmod>
+        <priority>0.85</priority>
+    </url>
```

Comment:
@SarahDonvito aggiungerei anche come seconda entry di login
Url: cittadini.notifichedigitali.it/auth/login